documentation/assemblies/managing/assembly-working-with-resources.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in:
+//
+// assembly-management-tasks.adoc
+
+[id="working-with-resources-{context}"]
+= Working with custom resources
+
+You can use `kubectl` commands to retrieve information and perform other operations on Strimzi custom resources.
+
+Using `kubectl` with the `status` property of a custom resource allows you to publish information about the resource to users and tools that need it.
```

Comment:
I do not think this sounds good. There are two points which can be confusing:
* `Using `kubectl` with the `status` property` is confusing what does property mean. I think we should use `subresource`
* `allows you to publish information` I do not think it lets you publish anything. You can only get the information

Do maybe

```suggestion
Using `kubectl` with the `status` subresource of a custom resource allows you to get the information about the resource.
```